src/main/java/com/ays/user/service/impl/UserSelfServiceImpl.java

```diff
@@ -52,7 +68,7 @@ public void updateUserSupportStatus(UserSupportStatusUpdateRequest updateRequest
     /**
      * Checks if a user has an assignment when update supportStatus to specific statuses.
      *
-     * @param userId the id of the user to check
+     * @param userId            the id of the user to check
```

Comment:
Buradaki tabı silebiliriz.

Reply:
Format aşağıdaki gibi olduğu için IDE kendisi hizalıyor, özellikle yapılan bir tab değil;
![image](https://github.com/afet-yonetim-sistemi/ays-be/assets/61833677/c5336dcc-458f-4cfb-8664-1ddc4031ef72)